sizing.tex

```diff
@@ -83,7 +105,7 @@ \subsubsection{Data Product Sizing}
 The size of this dataset was measured on disk; it is 2,564,358 CCD images, each at 18.2~MB (approximately three times the size of PDR1 alone).
 
 Output sizes are assumed to scale linearly with input size, and by the same factor for each instrument, except for coadds which scale by the sky area processed.
-While the Object catalog ought to be proportional to sky area as well, its size is expected to be dominated by Source and ForcedSource, so we conservatively make them all proportional to input size (visits).
+While the Object catalog ought to be proportional to sky area as well, its size is expected to be dominated by Source and ForcedSource, so we conservatively make them all proportional to input size (visits) for the precursor data where we do not have object count estimates.  For LSSTCam, we use the catalog row estimates to derive Qserv table sizes, but the Parquet file sizes are scaled based on HSC, as they may differ from the Qserv schema.
```

Comment:
For consistency with the rest of the document, you probably want a line-break in there.